cms/envs/common.py

```diff
@@ -2934,3 +2934,11 @@ def _should_send_learning_badge_events(settings):
 # .. for now it wil impact country listing in auth flow and user profile.
 # .. eg ['US', 'CA']
 DISABLED_COUNTRIES = []
+
+# .. setting_name: LIBRARY_ENABLED_BLOCKS
+# .. setting_default: ['problem', 'video', 'html', 'drag-and-drop-v2']
+# .. setting_description: List of block types that are enabled to be created/used
+# .. in libraries. Both basic blocks and advanced blocks can be included.
+# .. In the future, advanced block configuration will be handled individually per library.
+# .. eg ['problem', 'video', 'drag-and-drop-v2]
```

Comment:
```suggestion
```
The default is a good enough example

Reply:
Deleted: https://github.com/openedx/edx-platform/pull/36292/commits/c3336cc555c11554c9d8bfcf6e12d18d76e3ef6d#diff-a16fd1c6cf75bdebdf97a2633b533d39e43975c6e98af58787ed6af1282eca71L2943